maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractRewritePomsPhase.java

```diff
@@ -199,9 +201,15 @@ private void transform( ReleaseDescriptor releaseDescriptor, ReleaseEnvironment
     {
         result.setStartTime( ( startTime >= 0 ) ? startTime : System.currentTimeMillis() );
 
+        URI root = ReleaseUtil.getRootProject( reactorProjects ).getBasedir().toURI();
+
         for ( MavenProject project : reactorProjects )
         {
-            logInfo( result, "Transforming '" + project.getName() + "'..." );
+            URI pom = project.getFile().toURI();
+            logInfo( result,
+                     "Transforming " + root.relativize( pom ).getPath() + ' '
```

Comment:
I believe that `URI` is not necessary here, but `Path` can make it relative too, no?

Reply:
I suppose going through a Path instead of an URI gives the exact same code/complexity/efficiency

Reply:
True, but `URI` has its problems and `Path` is a new solid API.